docs/plugins/deploy.md

```diff
@@ -0,0 +1,35 @@
+# Deploy
+
+```
+serverless deploy --stage dev --region us-east-1
+```
+
+Deploys your service to AWS.
```

Comment:
Imho keep this one provider independent and we should add an AWS deployment documentation page that talks about the details of how AWS deployment works